R/utils-shinyrems.R

```diff
@@ -0,0 +1,80 @@
+#' get EMS data lookup table
+#'
+#' This function generates and caches a lookup table of the chosen data ('2yr' or '4yr') and imports it into your R session.
+#' If the cached data are more current than than the lookup cache, you will be prompted to update it.
+#'
+#' @param which Defaults to \code{"2yr"} (past 2 years). You can also specify "4yr"
+#' to get the past four years of data.
+#' @param ask should the function ask for your permission to cache data on your computer?
+#' Default \code{TRUE}
+#' @return a data frame
+#' @details this function is only used when running the ShinyRems app.
+#'
+#' \code{"EMS_ID", "MONITORING_LOCATION", "PERMIT", "PARAMETER_CODE", "PARAMETER",
+#' "LONGITUDE" "LATITUDE", "FROM_DATE", "TO_DATE"}
+#' @export
```

Comment:
Suggest adding `internal` keyword to documentation. This means that while you can still get the help for the function using `?get_ems_lookup` it won't be added to the package documentation index, which I would prefer since it's not meant for general use.

```suggestion
#' @export
#' @keywords internal
```